app/views/registrations/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<html class="Reg_Html">
+<body class="Reg_Body">
+<div class="container">
+  <div class="row">
+    <div class="col">
+    </div>
+  </div>
+</div>
+<%= form_with url: home_pages_index_url, method: :push do |form| %>
+  <main class="Reg_Main Reg_container">
+    <div class="container">
+      <div class="row">
+        <div class="col Reg_AboutYou">
+          <b class="Reg_b_About">About</b>
+          <b class="Reg_b_You">You</b>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <h3>
+            <b>Contact data:</b>
+          </h3>
+        </div>
+      </div>
+      <%= render "userInput" %>
+      <%= render "alerts" %>
+      <div class="form-check form-switch">
+        <input class="form-check-input" type="checkbox" id="Reg_ShowPassword">
+        <label class="form-check-label" for="Reg_ShowPassword">Show Password</label>
+      </div>
+      <%= render "userAgreement" %>
+      <div class="row">
+        <div class="col">
+          <%= form.submit "Registierung", id: "regBtn", class: 'btn btn-primary' %>
+        </div>
+      </div>
+    </div>
+<% end %>
+</main>
+</body>
+<%= javascript_pack_tag "Registration" %>
```

Comment:
Das gehört in Layout. Du hast praktisch mehrfach dein JS eingebunden und damit dein JS-Code mehrfach ausgeliefert.

Reply:
Ich habe jetzt die Javascript der Application.html.erb hinzugefügt